test/keypair_test.exs

```diff
@@ -1,40 +1,61 @@
 defmodule KeyPairTest do
   alias Elixium.KeyPair
-  use ExUnit.Case, async: true
 
-  setup _ do
-    on_exit(fn -> File.rm_rf!(".keys") end)
-    :ok
-  end
+  use ExUnit.Case, async: true
 
-  test "can create a keypair" do
+  test "Can create a keypair and return correct format private and public keys" do
     {pub, priv} = KeyPair.create_keypair()
 
     assert is_binary(pub)
     assert is_binary(priv)
   end
 
-  test "can create a signature" do
+  test "Key pair is generated and saved" do
+    path = Application.get_env(:elixium_core, :unix_key_address)
+    {public, private} = KeyPair.create_keypair
+    compressed_pub_address = KeyPair.address_from_pubkey(public)
+    key_path = path <> "/" <> compressed_pub_address <> ".key"
+    exists? = File.exists?(key_path)
+
+    assert exists? == true
+    with true <- assert exists? do
```

Comment:
Is this double assertion intentional?

Reply:
Yeah, until i get the test environment config running i have it deleting the files on a truthy result